chart/templates/server-ide-configmap.yaml

```diff
@@ -46,33 +46,22 @@ options:
     orderKey: "00"
     title: "VS Code"
     type: "browser"
+    label: "Browser"
     logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscode.svg"
+    # latestLogo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
     image: {{ (include "stable-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
-  code-latest:
-    orderKey: "01"
-    title: "VS Code"
-    type: "browser"
-    logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
-    tooltip: "Early access version, still subject to testing."
-    label: "Insiders"
-    image: {{ (include "insider-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
-    resolveImageDigest: true
+    latestImage: {{ (include "insider-image-full" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.codeImage)) }}
 
   # Desktop IDEs
   code-desktop:
     orderKey: "02"
     title: "VS Code"
     type: "desktop"
+    label: "Desktop"
     logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscode.svg"
+    # latestLogo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
     image: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktop)) }}
-  code-desktop-insiders:
-    orderKey: "03"
-    title: "VS Code"
-    type: "desktop"
-    logo: "https://ide.{{ $.Values.hostname }}/image/ide-logo/vscodeInsiders.svg"
-    tooltip: "Visual Studio Code Insiders for early adopters."
-    label: "Insiders"
-    image: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktopInsiders)) }}
+    latestImage: {{ (include "gitpod.comp.imageFull" (dict "root" $ "gp" $gp "comp" $gp.components.workspace.desktopIdeImages.codeDesktopInsiders)) }}
   intellij:
     orderKey: "04"
     title: "IntelliJ IDEA"
```

Comment:
In fact, the client IDE is JetBrains [Gateway](https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway), maybe we can ask @loujaybee for more details

Reply:
OK, let's wait to see what @loujaybee thinks. 🏓 

**thought**: The button could be better if describing the action to be performed which in this case eventually is to open the editor not the gateway. For users, opening the gateway is a secondary and intermediate step required to accomplish the job to be done. Hiding the complexity of this extra step can help users and could also decrease confusion in regards to the performing action of the button.

Reply:
A good thought, @gtsiolis and I agree  💡 

> out of the scope of the changes in this PR

But yeah, out of scope, unless @mustard-mh wants to / can incorporate here. 

> the client IDE is JetBrains [Gateway](https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway)

It would be fine to refer to the direct IDE, e.g. "_Open IntelliJ IDEA_" as @gtsiolis originally suggested... If we're being pedantic 🤓 it opens Gateway, which then downloads a thin client for your existing IDE backend e.g. IntelliJ IDEA 😁 

If it's an easy change, feel free to add @mustard-mh, or we can open a separate issue 🙏

Reply:
@mustard-mh If making the button label changes sounds more complicated let's open a follow up isue for this. 🏓

Reply:
It's a bug 🐛  here https://github.com/gitpod-io/gitpod/pull/8783/files#diff-f7dcb26962daf6bfdfa64d5694b28c680f2c2832943a8c6e858ba10fa50c1491R11 which $3 is treated as label or button https://github.com/gitpod-io/gitpod/blob/bb210362ef2bff0354fe57419c7ffe5793cb76e3/components/ide/jetbrains/image/status/main.go#L33-L36

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/9201 to track this discussion.